bin/check_metrics.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+#
+import sys
+from prometheus_api_client import PrometheusConnect
+from prometheus_api_client.utils import parse_datetime
+from datetime import timedelta
+from datetime import datetime
+
+url=sys.argv[1]
+time_at=sys.argv[2]
+period=sys.argv[3]
+
+prom=PrometheusConnect(url=url, disable_ssl=True)
+
+def error_exit(info:str):
+    print(info)
+    sys.exit(1)
+
+for metric_name in ['emqx_client_subscribe', 'emqx_messages_publish', 'emqx_client_connected', 'emqx_connections_count']:
+
+    # 1) check: all traffic counters are non-zero for last x mins
+    print(f"checking {metric_name}")
+    query_str="sum(increase(%s[%s]))" % (metric_name, period)
+    # example: [{'metric': {}, 'value': [1644946184, '1578.9473684210525']}]
```

Comment:
Just curiosity: what are the 2 values in `value`?

It seems like index 0 is used [below](https://github.com/emqx/cdk-emqx-cluster/pull/66/files#diff-320ab3dd089b2e563504f7432a533d48d71dc7c08d29857b5e721d9ceca05dc7R27) as the count, but it has the looks of a timestamp rather than the sum itself?

Reply:
yes, 1644946184 is timestamp :+1: